terraform/stacks/cognito_aai/app_status_page.tf

```diff
@@ -14,15 +14,18 @@ locals {
     stg  = ""
   }
 
+  # FIXME: remove this when status page and apis is ready in production, https://github.com/umccr/data-portal-status-page/issues/105
+  status_page_dev_domain = "status.${var.base_domain[terraform.workspace]}"
```

Comment:
Hey Ray; it is okay to update `status_page_domain` at line 9 as well as line 12. Instead of introducing this temporary var.

We have terraform workspace isolated. So, This is fine.

Reply:
sure,  will make the change now